packages/rx_bloc_cli/mason_templates/bricks/feature_google_login/__brick__/lib/lib_auth_google/di/lib_auth_google_page_with_dependencies.dart

```diff
@@ -0,0 +1,63 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_rx_bloc/flutter_rx_bloc.dart';
+import 'package:google_sign_in/google_sign_in.dart';
+import 'package:provider/provider.dart';
+
+import '../../base/data_sources/remote/http_clients/api_http_client.dart';
+import '../blocs/google_login_bloc.dart';
+import '../data_sources/remote/google_auth_data_source.dart';
+import '../repositories/google_auth_repository.dart';
+import '../services/google_login_service.dart';
+import '../ui_components/lib_auth_google_button.dart';
+
+class LibAuthGooglePageWithDependencies extends StatelessWidget {
+  const LibAuthGooglePageWithDependencies({Key? key}) : super(key: key);
+
```

Comment:
Update class and file names regarding the widget name. See the comment below for the widget. Example: GoogleLoginButton / GoogleLoginButtonWithDependencies